src/java.base/share/classes/jdk/internal/util/DecimalDigits.java

```diff
@@ -335,70 +335,6 @@ public static int getCharsUTF16(long i, int index, byte[] buf) {
         return charPos;
     }
 
-    /**
-     * This is a variant of {@link DecimalDigits#getCharsUTF16(long, int, byte[])}, but for
-     * UTF-16 coder.
-     *
-     * @param i     value to convert
-     * @param index next index, after the least significant digit
-     * @param buf   target buffer, UTF16-coded.
-     * @return index of the most significant digit or minus sign, if present
-     */
-    public static int getChars(long i, int index, char[] buf) {
```

Comment:
Before dropping this method, there is another candidate to use  it here : https://github.com/openjdk/jdk/blob/1d2eb2fbaea700fc77b644b5eb5a8a7c40ede108/src/java.base/share/classes/jdk/internal/math/FloatingDecimal.java#L367

Reply:
I took a look at the FloatingDecimal related code. The relevant code in FloatingDecimal can be refactored and simplified so that the removed code does not need to be used.

I submitted a PR to simplify FloatingDecimal https://github.com/openjdk/jdk/pull/23311